core/src/apps/cardano/layout.py

```diff
@@ -195,18 +199,37 @@ async def show_address(
     Custom show_address function is needed because cardano addresses don't
     fit on a single screen.
     """
-    path_str = address_n_to_str(path)
-    t1 = Text(path_str, ui.ICON_RECEIVE, ui.GREEN)
+
+    address_type_label = "%s address" % ADDRESS_TYPE_NAMES[address_type]
+    t1 = Text(address_type_label, ui.ICON_RECEIVE, ui.GREEN)
```

Comment:
Please rename `t1` to something else, it evokes Trezor One to me :).

Reply:
I realized this pattern is used accross the whole `layout.py` and I agree the naming is misleading, so I refactored it to `page<number` in the whole file